src/tests/unit/services/VerifiableCredentialService.test.ts

```diff
@@ -0,0 +1,153 @@
+import { UserInfoRequestProcessor } from "../../../services/UserInfoRequestProcessor";
+import { Metrics } from "@aws-lambda-powertools/metrics";
+import { mock } from "jest-mock-extended";
+import { Logger } from "@aws-lambda-powertools/logger";
+import { BavService } from "../../../services/BavService";
+import { Response } from "../../../utils/Response";
+import { HttpCodesEnum } from "../../../models/enums/HttpCodesEnum";
+import { ISessionItem } from "../../../models/ISessionItem";
+import { PersonIdentityItem } from "../../../models/PersonIdentityItem";
+import { absoluteTimeNow } from "../../../utils/DateTimeUtils";
+import {
+	MockKmsJwtAdapterForVc,
+} from "../utils/MockJwtVerifierSigner";
+import { VALID_VC } from "../data/verified_credential";
+import { VALID_USERINFO } from "../data/userInfo-events";
+import { Constants } from "../../../utils/Constants";
+import { isValidUUID } from "../../../utils/Validations";
```

Comment:
A fair few of these imports aren't needed

Reply:
Tided up thanks